recipes/nmos-cpp/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.1)
+project(NmosCppTestPackage CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
```

Comment:
```suggestion
conan_basic_setup(TARGETS)
```

Reply:
Does requesting `TARGETS` just ask for the `CONAN_PKG::` targets? I didn't think this CMakeLists.txt used those.

Reply:
When not passing `TARGETS`, `conan_basic_setup` does `include_directories`, which will make that the targets are not fully tested.

Reply:
👍 Thanks, makes sense.